src/battle_script_commands.c

```diff
@@ -3801,6 +3801,53 @@ void SetMoveEffect(bool32 primary, u32 certain)
                     gBattlescriptCurrInstr = BattleScript_SpikesActivates;
                 }
                 break;
+            case MOVE_EFFECT_TRIPLE_ARROWS:
+                {
+                    u8 randomLowerDefenseChance;
+                    u8 randomFlinchChance;
+
+                    if (GetBattlerAbility(gBattlerAttacker) == ABILITY_SERENE_GRACE)
+                    {
+                        randomLowerDefenseChance = RandomPercentage(RNG_TRIPLE_ARROWS_DEFENSE_DOWN, 100);
+                        randomFlinchChance = RandomPercentage(RNG_TRIPLE_ARROWS_FLINCH, 60);
```

Comment:
Set the base percentages as 30 and 50 and then double those for Serene Grace. Don't hardcode 30, 50, 60, and 100.

Reply:
By "set the base percentages", do you mean in the move's data or...?

Reply:
Eg.
```
lowerDefenseChance = 50
flinchChance = 30
if (DoubledSecondaryEffectChance(gBattlerAttacker))
{
    lowerDefenseChance *= 2
    flinchChance *= 2
}

randomLowerDefenseChance = RandomPercentage(RNG_TRIPLE_ARROWS_DEFENSE_DOWN, lowerDefenseChance);
randomFlinchChance = RandomPercentage(RNG_TRIPLE_ARROWS_FLINCH, flinchChance);
```